site/specs/numbers.json

```
@@ -15679,68 +15838,66 @@
+                  "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<TnOptionOrder>\n  <CustomerOrderId>TnOptionOrder1</CustomerOrderId>\n  <TnOptionGroups>\n    <TnOptionGroup>\n      <NumberFormat>10digit</NumberFormat>\n      <RPIDFormat>10digit</RPIDFormat>\n      <RewriteUser>testUser1</RewriteUser>\n      <CallForward>6042661720</CallForward>\n      <CallingNameDisplay>on</CallingNameDisplay>\n      <PortOutPasscode>a1b2c3</PortOutPasscode>\n      <Protected>true</Protected>\n      <Sms>on</Sms>\n      <FinalDestinationURI>sip:+12345678901@1.2.3.4:5060</FinalDestinationURI>\n      <A2pSettings>\n        <MessageClass>M</MessageClass>\n        <CampaignId>campaignId010</CampaignId>\n        <Action>asSpecified</Action>\n      </A2pSettings>\n      <TelephoneNumbers>\n        <TelephoneNumber>2018551020</TelephoneNumber>\n      </TelephoneNumbers>\n    </TnOptionGroup>\n    <TnOptionGroup>\n      <CallingNameDisplay>off</CallingNameDisplay>\n      <Protected>false</Protected>\n      <Sms>off</Sms>\n      <OriginationRoutePlan>\n        <Route>\n          <Endpoint>6105552502</Endpoint>\n          <Priority>1</Priority>\n          <Weight>100</Weight>\n        </Route>\n        <Route>\n          <Endpoint>sip:clarkkent@dailyplanet.com</Endpoint>\n          <Priority>2</Priority>\n          <Weight>100</Weight>\n        </Route>\n        <Route>\n          <Endpoint>192.168.20.21</Endpoint>\n          <Priority>3</Priority>\n          <Weight>50</Weight>\n        </Route>\n        <Route>\n          <Endpoint>7075552509</Endpoint>\n          <Priority>3</Priority>\n          <Weight>50</Weight>\n        </Route>\n        <Route>\n          <Endpoint>myhost.bandwidth.example</Endpoint>\n          <Priority>4</Priority>\n          <Weight>100</Weight>\n        </Route>\n        <Action>asSpecified</Action>\n      </OriginationRoutePlan>\n      <TelephoneNumbers>\n        <TelephoneNumber>2018551025</TelephoneNumber>\n      </TelephoneNumbers>\n    </TnOptionGroup>\n    <TnOptionGroup>\n      <PindropEnabled>true</PindropEnabled>\n      <TelephoneNumbers>\n        <TelephoneNumber>2018551026</TelephoneNumber>\n      </TelephoneNumbers>\n    </TnOptionGroup>\n  </TnOptionGroups>\n</TnOptionOrder>"
                 }
               }
             }
           }
         },
         "responses": {
           "201": {
-            "description": "<p>Created <br> A 201 response indicates that an order has been created.</p>\n",
+            "description": "Created A 201 response indicates that an order has been created.",
```

```suggestion
            "description": "Created",
```
Just `Created` is fine